app/views/idv/inherited_proofing/verify_info.html.erb

```diff
@@ -0,0 +1,10 @@
+<% title t('inherited_proofing.headings.lets_go') %>
+
+<h2>Place holder to show Please verify your information</h2>
+
+<%= simple_form_for :inherited_proofing,
+                    url: url_for,
+                    method: 'put',
+                    html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
+    <%= f.submit t('inherited_proofing.buttons.continue') %>
```

Comment:
Seems like you could make line 8 and 9, within the same <% %> block... not sure?

Reply:
@jscodefix Just following the same patterns from get-started and agreement page.
https://github.com/18F/identity-idp/blob/d25131b5ee084745043c7476987874dcaf465d28/app/views/idv/inherited_proofing/get_started.html.erb#L44

Reply:
OK, I tested my idea, and it does not work.  I also now see that Gene was the one that added the I18n tests... ugg.